third_party/move/tools/move-linter/tests/model_ast_lints/assert_const_warn.move

```diff
@@ -0,0 +1,59 @@
+module 0xc0ffee::m {
+    public fun test1_warn() {
+        assert!(true);
```

Comment:
Needs test cases for:
- asserts with a code, like: `assert!(true, 55);`
- asserts with a const bool value, like: `assert!(BOOL_CONST1)` (where `const BOOL_CONST1: bool = true;`).

Maybe have a case where there is a function with a `#[test]` attribute?